pom.xml

```diff
@@ -261,7 +261,7 @@
           <extensions>true</extensions>
           <configuration>
             <instructions>
-              <_exportcontents>co.cask.*;org.apache.spark.streaming.kafka.*;
+              <_exportcontents>io.cdap.plugin.filemerge.*;org.apache.spark.streaming.kafka.*;
                 kafka.serializer.*;kafka.common;org.apache.kafka.*;com.mapr.streams.*;
```

Comment:
these other exports look like they were copied over from another repo, please remove them. It should only need io.cdap.plugin.filemerge.*

Reply:
done